FEXCore/Source/Interface/Core/ArchHelpers/CodeEmitter/SVEOps.inl

```diff
@@ -1384,6 +1384,12 @@ public:
   }
 
   // SVE predicate initialize
+  void ptrue(SubRegSize size, PRegister pd, PredicatePattern pattern) {
```

Comment:
We should probably drop the templated ones if these exist, since they don't really buy us anything that these wouldn't

Reply:
Removed the templated versions and switched the dispatcher over to it.